sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+sonar.projectKey=your_project_key
+sonar.organization=your_organization_key
+sonar.host.url=https://sonarcloud.io
+sonar.login=your_sonarcloud_token
+sonar.exclusions=**/src/**/__tests__/**
```

Comment:
important: only `sonar.exclusions` is needed, the rest is handled by the environment variables.

Reply:
That's a draft but indeed I was wondering how to pass these variables to Sonar. I tried by specifying only sonar.exclusions but did not work properly